packages/editor/templates/Hello World/README.md

```diff
@@ -1,198 +1,95 @@
-# Hello World template
+# Hello World
 
-This template is a "Hello World" example that teaches you how to:
+The goal of this "Hello World" project template is to teach you how to:
 
--   Provide arguments to a contract constructor using the _Configure contract_ modal.
--   Store state in a contract and to update it.
--   Fetch your newly created contract's information from the blockchain and render it to a front end.
+-   Deploy an [Ethereum smart contract](https://ethereum.org/learn/#smart-contracts) written in the [Solidity programming language](https://ethereum.org/developers/#smart-contract-languages).
+-   Fetch your contract's state from the blockchain and render it to a frontend using a [JavaScript library](https://ethereum.org/developers/#frontend-javascript-apis).
+-   Update state variables of your deployed contract by interacting with your app in the IDE's Browser.
 
-> Tutorial content supplied by [kauri.io](https://kauri.io).
+If you'd like to learn more about how Ethereum works before diving in, we recommend you [start here](https://ethereum.org/learn/).
```

Comment:
Yup, we can also remove this from other templates, Kauri is now over anyway

Reply:
RIP 😕 you guys did great stuff!